ivadomed/training.py

```diff
@@ -54,6 +55,22 @@ def train(model_params, dataset_train, dataset_val, training_params, path_output
     # Write the metrics, images, etc to TensorBoard format
     writer = SummaryWriter(log_dir=path_output)
 
+    # Only initialize wandb if required params are found in the config file
+    if wandb_params is not None and wandb_params[WandbKW.WANDB_API_KEY]:
+        # Collect all hyperparameters into a dictionary
+        cfg = { **training_params, **model_params}
+
+        # Get the actual project, group, and run names if they exist, else choose the temporary names as default
+        project_name = wandb_params.get(WandbKW.PROJECT_NAME, "temp_project")
+        group_name = wandb_params.get(WandbKW.GROUP_NAME, "temp_group")
+        run_name = wandb_params.get(WandbKW.RUN_NAME, "temp_run")
```

Comment:
I would also recommend maybe give an output to user to mention that. Because otherwise, they won't look for `temp_project`/`temp_group`/`temp_run` etc.

Reply:
Yes, good point! I will put out a print statement here with those names